components/DetailPullup.tsx

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import { StyleSheet, Image, FlatList } from "react-native";
+import { View, Text } from "./Themed";
+
+const item = {
+  title: "Blue Pants",
+  user: "ravina patel",
+  description: `
+  Vintage blue pants that are super comfy and cool!
+  Condition
+  ISBN
+  Additional Information
+  `,
+  price: 25.00
+}
+
+const similarItems: string[] = [
+  "rainbow pants",
+  "clear pants",
+  "khaki pants",
+  "jean pants",
+  "short pants",
+  "long pants",
+  "pant legs",
+  "fancy pants"
+]
+
+export default function DetailPullUp() {
+  return (
+    <View style={styles.container}>
+      <View style={styles.expandRow}>
+        <Text style={styles.title}>{item.title}</Text>
+        <Text style={styles.price}>{"$ " + item.price + ".00"}</Text>
```

Comment:
Why are you adding `.00` to the price here when your price (line 14) already has two decimal places? Also adding this manually could result in incorrect price tags when the price is not a whole number (e.g. 9.99)